tests/solrand.ts

```diff
@@ -71,16 +72,13 @@ describe('solrand', () => {
         assert(requester.account.count.toNumber() == 1); // Before 0, now 1
         assert(requester.account.authority.toString() == userKeypair.publicKey.toString()); // Still the same owner
         assert(requester.account.activeRequest); // Now true
-
-        console.log(`Cost of request is ${beforeBalance - afterBalance}`);
-        assert((newOracleBalance - oldOracleBalance) == FEE);
     });
 
     it('Cannot make multiple requests in a row', async () => {
```

Comment:
Should we add a test to assert that you can create multiple random requests with different uuids?